### PR TITLE
Master fix pay directly las

### PR DESCRIPTION
**Pull request by qdp-odoo**
on behalf of @smetl 

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account_voucher/models/account_voucher.py [hunk 1]
```diff
@@ -101,6 +112,15 @@ def name_get(self):
     def _get_journal_currency(self):
         self.currency_id = self.journal_id.currency_id.id or self.company_id.currency_id.id
 
+    @api.depends('company_id')
+    def _compute_payment_journal_id(self):
```
Comment (qdp-odoo) [inline addons/account_voucher/models/account_voucher.py]: dead code to remove

File: addons/account_voucher/models/account_voucher.py [hunk 2]
```diff
@@ -226,20 +235,29 @@ def _convert_amount(self, amount):
...
+            payment_methods = self.journal_id.inbound_payment_method_ids
+            payment_type = 'inbound'
+            partner_type = 'customer'
+            sequence_code = 'account.payment.customer.invoice'
+        else:
+            payment_methods = self.journal_id.outbound_payment_method_ids
+            payment_type = 'outbound'
+            partner_type = 'supplier'
+            sequence_code = 'account.payment.supplier.invoice'
+        name = self.env['ir.sequence'].with_context(ir_sequence_date=self.date).next_by_code(sequence_code)
```
Comment (qdp-odoo) [inline addons/account_voucher/models/account_voucher.py]: same remark as in for the stable PR

File: addons/account_voucher/models/account_voucher.py [hunk 3]
```diff
@@ -316,6 +331,16 @@ def action_move_line_create(self):
...
 
+            # Create a payment to allow the reconciliation when pay_now = 'pay_now'.
+            if self.pay_now == 'pay_now' and self.amount > 0:
+                payment_id = self.env['account.payment'].create(self.voucher_pay_now_payment_create())
+                payment_id.post()
+
+                # Reconcile the receipt with the payment
+                (payment_id.move_line_ids + move.line_ids)\
+                    .filtered(lambda l: l.account_id == self.account_id)\
+                    .reconcile()
```
Comment (qdp-odoo) [inline addons/account_voucher/models/account_voucher.py]: arf, it's correct but it hurts my eyes... use a variable and write that in 2 one-liners

File: addons/account_voucher/views/account_voucher_views.xml [hunk 4]
```diff
@@ -270,6 +274,7 @@
                         </page>
                         <page string="Journal Items" groups="account.group_account_user">
                             <group>
+                                <field name="account_date" groups="base.group_no_one"/>
```
Comment (qdp-odoo) [inline addons/account_voucher/views/account_voucher_views.xml]: same remark as in the stable PR

File: addons/account_voucher/views/account_voucher_views.xml [hunk 5]
```diff
@@ -387,6 +393,7 @@
...
```
Comment (qdp-odoo) [inline addons/account_voucher/views/account_voucher_views.xml]: same remark as in the stable PR

File: addons/account_voucher/models/account_voucher.py (hunk 1 above)
Comment (smetl) [reply addons/account_voucher/models/account_voucher.py]: It seems my rebase did something bad...

File: addons/account_voucher/views/account_voucher_views.xml (hunk 5 above)
Comment (smetl) [reply addons/account_voucher/views/account_voucher_views.xml]: same comment as in the previous "same remark as in the stable PR" remark.
It seems you are doing some remark-ception in this task!

Comment (qdp-odoo) [reply addons/account_voucher/views/account_voucher_views.xml]: :smile: 

it's you: it looks like you are doing some remark-able effort to comply with Laura's demand!

just remove the groups="base.group_no_one" and I'll be happy

Comment (smetl) [reply addons/account_voucher/views/account_voucher_views.xml]: You can be happy now! :wink:

Comment (qdp-odoo) [reply addons/account_voucher/views/account_voucher_views.xml]: :hugs:

**Comment by qdp-odoo** [comment]
landed in master, for saas 11.3, at revision https://github.com/odoo/odoo/commit/2ded1015d69ed526610d0a4ed26a5e4d4d880e7b

**Pull request closed without merging**